src/initialinput.py

```diff
@@ -0,0 +1,35 @@
+
+def prompt_for_input(menu, prompt):
+    '''"Request user input based on current runmode'''
+    print(menu)
+    return input(prompt)
+
+
+
+
+    if runmode == 1:
+        # main input
+        prompt_for_input(menu, prompt)
+        menu = ''What would you like to do:
+                            \t[1] Print a Report of donors and donation history.
+                            \t[2] Send a Thank You email for a new donation.
+                            \t[3] Quit the Program"
```

Comment:
This menu appears to me to be a syntax error.  I see two single quote marks on line 13, and a double-quote on line 16.  I *think* that you are trying to use a multi-line string.  That's defined as a string with triple-quotes before and after.  This is neither.